release/release_build.py

```diff
@@ -255,6 +266,8 @@ def build_checker_framework_release(version, afu_version, afu_release_date, chec
     return
 
 def commit_to_interm_projects(jsr308_version, afu_version, projects_to_release):
+    """Commit the changes for each project from its build repo to its corresponding intermediate repo"
+    "in preparation for running the release_push script, which does not read the build repos."""
```

Comment:
The single quotes at the end of the previous line and the beginning of this one look extraneous.  Python doesn't require them within a triple-quoted string.

It's good to break the lines, and all other very long lines (> 100 or >80 characters) that were added in this pull request should also be broken for readability, both in the GitHub pull request and in the final code.